### ISSUE TITLE
How to use the data train LLMs?

### ISSUE THREAD

**Issue by lei1993** [tangqiaoyu/ToolAlpaca#4]

**Comment by tangqiaoyu** [comment]
I apologize for the delayed response. To utilize the data for training LLMs, you would need to design a custom prompt. In our case, we adopted a prompt structure similar to [LangChain MRKL](https://github.com/langchain-ai/langchain/blob/master/libs/langchain/langchain/agents/mrkl/prompt.py). We then perform standard instruction finetuning on Vicuna models and only optimize the loss on `thought`, `action`, and `action input` parts.
